standard.tex

```diff
@@ -0,0 +1,199 @@
+  $g^z = h g^{H(t)} \Rightarrow g^z = g^{y + H(t)} \Rightarrow y \equiv z - H(t) \Modulo{q}$, so $\mathcal{A}^*$ is successful.
+  Therefore $\Pr[\mathcal{A}^*(h) = y] = p$.
+  But $\Pr[\mathcal{A}^*(h) = y]$ is negligible.
+\end{proof}
+
+This observation illustrates the useful fact that, if a single group element with unknown logarithm is provided, an arbitrary number of such group elements can be found and logarithm ignorance can be proven.
+
+\noindent\textbf{Proofs-of-ignorance.}
+There are other constructions which can give similar results. In fact, recent
+work on \emph{proofs-of-ignorance}~\cite{deshpande2018proofs} has shown that any
+$\NP$ language can support proofs-of-ignorance, which are a prerequisite for our
+need of unspendability (as inability to produce signatures mandates ignorance of
+the private key). Therefore, we conjecture that such constructions
+are possible using any secure signature scheme in which the secret key
+constitutes a witness for the fact that the public key is an element of an $\NP$
+language. Additionally, they argue that such constructions are not possible in
+the standard model given non-uniform probabilistic polynomial-time adversaries,
+supporting our construction in the Common Random String model. Whether burn
+constructions in the Standard Model exist against uniform probabilistic
+polynomial-time adversaries remains a question for future work.
+
+\import{./}{algorithms/alg.collision-adversary-crs.tex}
+
+\begin{theorem}[Binding]
+  If the hash function $H$ is collision resistant and its range lies in $[q]$ where $q$ denotes the group order of $\mathbb{G}$, then the Proof of Burn protocol $\Pi$ of Algorithm~\ref{alg.construction-crs} is \emph{binding}.
+\end{theorem}
+\begin{proof}
+  Let $\mathcal{A}$ be a probabilistic polynomial-time binding adversary against the protocol $\Pi$. We construct the probabilistic polynomial-time collision adversary $\mathcal{A}^*$ against the hash function $H$. The adversary $\mathcal{A}^*$ is illustrated in Algorithm~\ref{alg.collision-adversary-crs} and works as follows. It invokes $\mathcal{A}$ which returns a triplet $(t, t', \burnAddr)$, then returns the collision $(t, t')$. Let $p$ denote the probability that $\mathcal{A}$ is successful.
+
+  Conditioned on the event that $\mathcal{A}$ is successful, it holds that
+  $h g^{H(t)} = h g^{H(t')}$ and $t \neq t'$. This implies that $g^{H(t)} = g^{H(t')}$, which in turns yields $H(t) \equiv H(t') \Modulo{q}$. Since the range of $H$ lies in $[q]$, this constitutes a collision and $\mathcal{A}^*$ is successful.
+
+  We thus conclude that $\mathcal{A^*}$ is successful in the $\collisionattack$ game if and only if $\mathcal{A}$ is successful in the $\bindattack$ game.
+
+  \[
+    \Pr[\bindattack_{\mathcal{A},\Pi} = \true]
+    =
+    \Pr[\collisionattack_{\mathcal{A}^*,H} = \true]
+  \]
+
+  From the collision resistance of $H$ it follows that $\Pr[\textsf{Collision}_{\mathcal{A}^*,H} = 1] < \negl$. Therefore,
```

Comment:
Use \collisionattack and make the macro consistent with the other attack macros?

Reply:
one line above it was \true instead of 1